### PR TITLE
Handle nils properly when checking Description fields in controller test

### PR DESCRIPTION
**Pull request by luksa**
Sorry, I didn't handle this properly in my original commit.

### CODE REVIEW

File: pkg/controller/controller_test.go [hunk 1]
```diff
@@ -2295,8 +2297,10 @@ func TestCatalogConversionMultipleServiceClasses(t *testing.T) {
 		// For service1 make sure we have service level metadata with field1 = value1 as the blob
 		// and for service1 plan s1plan2 we have planmeta = planvalue as the blob.
 		if sc.Name == "service1" {
-			if *sc.Description != "service 1 description" {
-				t.Fatalf("Expected service1's description to be \"service 1 description\", but was: %s", sc.Description)
+			if sc.Description == nil {
+				t.Fatalf("Expected service1's description to be \"service 1 description\", but was nil")
+			} else if *sc.Description != "service 1 description" {
+				t.Fatalf("Expected service1's description to be \"service 1 description\", but was: %s", *sc.Description)
```
Comment (luksa) [inline pkg/controller/controller_test.go]: Is there a less verbose way of checking this?

**Comment by duglin** [comment]
LGTM

File: pkg/controller/controller_test.go (hunk 1 above)
Comment (vaikas-google) [inline pkg/controller/controller_test.go]: I _think_ this works:
if plan.Description == nil || *plan.Description != planDescription {
    t.Fatalf("Expected... 

But then the print will be wrong, so I am not aware (which of course doesn't mean there is not :) )

Comment (luksa) [inline pkg/controller/controller_test.go]: I know I could have used Fatalf("...%v", sc.Description), but didn't like the output too much. But then again, this is just a test, so it wouldn't be too bad, I guess.

Comment (duglin) [inline pkg/controller/controller_test.go]: stick with this - it works - let's get this merged :-)

**Pull request merged**